.travis.yml

```diff
@@ -9,10 +9,8 @@ matrix:
   include:
     - python: 3.7
       dist: xenial
```

Comment:
You might be able to move `dist: xenial` out one level and define it globally.  If so, you can move the python declarations out to the global list and eliminate the matrix include section all together.

Reply:
I can confirm that both solutions work. While xenial is generally available, it is not the Travis default so I usually take the more conservative route of only using it where it is needed.

Reply:
#831 eliminates one test when run on xenial so I think it is best to run more tests on Pythons that can run on the Travis CI default distro and fewer tests only on Python 3.7 and nightly.